app/components/search-results/template.html

```diff
@@ -8,7 +8,7 @@ <h2 ng-if="!search.documents.length">No results for <em>{{search.query}}</em></h
           <div class="ph-lg-margin-top media" ng-repeat="document in search.documents">
             <h4>
               <a href="./documents/{{document.id}}" title="{{document.title}}">
-                {{document.title}}
+                <kramjax body="document.title"></kramjax>
```

Comment:
I know of no document title that actually has a line break in it, so adding would probably no influence whatsoever. I've added it now, simply for consistency's sake. (Likewise below.)